ui/app/src/components/NewRemoteRepositoryDialog/NewRemoteRepositoryDialogContainer.tsx

```diff
@@ -47,7 +47,7 @@ export default function NewRemoteRepositoryDialogContainer(props: NewRemoteRepos
           : inputs.repoAuthentication === 'token'
           ? { remoteUsername: inputs.repoUsername, remoteToken: inputs.repoToken }
           : inputs.repoAuthentication === 'key'
-          ? { remotePrivateKey: inputs.repoToken }
+          ? { remotePrivateKey: inputs.repoKey }
```

Comment:
@jvega190 
please check if there are missing/too-loose typings. 
is repoToken an actual thing?

Reply:
@jvega190 👆🏻

Reply:
The other input props seem fine. `repoToken` is the value mapped to the git token used in the form for authentication